blockchain/contracts/PropertyOwnership.sol

```diff
@@ -14,7 +14,9 @@ contract PropertyOwnership is PropertyBase, ERC721Token {
   function buy(uint256 _propertyId) payable {
     require(msg.value >= properties[_propertyId].buy_price);
 
-    //
+    if (msg.value > properties[_propertyId].buy_price) {
+      msg.sender.send(msg.value - (properties[_propertyId].buy_price * 10 ** 18));
```

Comment:
Don't understand what this line is doing!

Reply:
It's converting the buy_price from ether to wei - we should pull it out into a variable, especially seeing as it should be being used on line 17 too.

Reply:
Can we extract it to a `pure` func?